Tests/Model/CardPackage/DeckTest.java

```diff
@@ -23,16 +23,16 @@
      * This resets the decks before every test
      ***********************************************************************************/
     @Before
-    void setUp(){
-        chanceDeck = new Deck(System.getenv("CHANCE_FILE"),true);
-        communityDeck = new Deck(System.getenv("COMMUNITY_FILE"),false);
+    public void setUp(){
+        chanceDeck = new Deck("res/chance.txt",true);
+        communityDeck = new Deck("res/community.txt",false);
```

Comment:
This changes aren't needed. The reason they are environmental variables it to prevent errors finding the files between machines.

Reply:
When I went to perform the test, I was getting a null pointer exception when trying to load the files. I had to define a path in order for it to work. is System.getenv a feature of java 9?